docs/book/cloud_provider_interface.md

```diff
@@ -102,4 +102,4 @@ CPI is supported for vSphere versions 6.7U3 or later. CPI also requires a Kubern
 
 ## How do I install it?
 
-See the [Deploying cloud-provider-vsphere docs](https://cloud-provider-vsphere.sigs.k8s.io/tutorials/deploying_cloud_provider_vsphere_with_rbac.html) for install steps.
```

Comment:
I think we want this to point to https://cloud-provider-vsphere.sigs.k8s.io/tutorials/kubernetes-on-vsphere-with-kubeadm.html going forward.